lib/scrabble.rb

```diff
@@ -15,4 +24,27 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  #multiply that value by the corresponding multiplier value, do an enumerable to cycle through array applying multiplier to each in the array using map
+  def multiply_word_with_letter_multiplier(word,letter_multiplier)
```

Comment:
I would've held off on this functionality until I made sure I was able to score a word with no multiplier.

1) score a single letter
2) score an empty string
3) return a 0 when someone passes `nil`
4) score a whole word
5) score with a letter multiplier array
6) score with a letter multiplier array and a word multiplier
7) score with a letter multiplier array, a word multiplier, and a 7-letter bonus

It looks like you were trying to tackle 5, and setting the foundations for 6 without completing 1-4. The unfortunate result of trying to build a lot of different things is that none of them was completed. It would've been better to see those first four working with some plans for 5-6.